multipatch_analysis/database/morphology.py

```diff
@@ -7,7 +7,18 @@
 
 Morphology = make_table(name='morphology', comment="Describes morphological properties of cells.", columns=[
     ('cell_id', 'cell.id', 'The ID of the cell described by each record', {'index': True, 'unique': True}),
-    ('pyramidal', 'bool', 'Whether the experimenter labeled this cell as pyramidal', {'index': True}),
+    ('cell_specimen_id', 'int', 'LIMS cell specimen ID', {'index': True}),
+    ('morpho_db_hash', 'int', 'hash of Morphology team database for update tracking', {'index': True}),
```

Comment:
No index here.

Reply:
what does the index do exactly?

Reply:
makes the column faster to search (which we don't need) and slower to write.